docs/csharp/whats-new/csharp-7.md

```diff
@@ -49,7 +49,7 @@ Finally, the compiler has new options:
 
 The remainder of this article provides an overview of each feature. For each feature, you'll learn the reasoning behind it and the syntax. You can explore these features in your environment using the `dotnet try` global tool:
 
-1. Install the [dotnet-try](https://github.com/dotnet/try/blob/master/README.md#setup) global tool.
+1. Install the [dotnet-try](https://github.com/dotnet/try/blob/main/README.md#setup) global tool.
```

Comment:
There is no "setup" heading. Should I link to just `https://github.com/dotnet/try`?

Reply:
It looks like the link should go here: https://github.com/dotnet/try/blob/main/DotNetTryLocal.md

Reply:
```suggestion
1. Install the [dotnet-try](https://github.com/dotnet/try/blob/main/DotNetTryLocal.md) global tool.
```